tests/build_image_test.py

```diff
@@ -91,7 +91,7 @@ def test_local_conda_channel_with_absolute_path(mocker):
     '''
     Test for build_runtime_image with local conda channel with its absolute path
     '''
-    intended_image_name = build_image.REPO_NAME + ":" + build_image.IMAGE_NAME
+    intended_image_name = build_image.REPO_NAME + ":" + build_image.IMAGE_NAME + "-" + "test-conda-env"
```

Comment:
Just a nit pick - Can we make this "test-conda-env" as constant variable which is declared once.

Reply:
That's probably a good idea. That makes it easy to change if we need to.

Reply:
I have added a commit to address this. @npanpaliya, @jayfurmanek please have a look.

Reply:
Similar change would be needed in other tests too. I will create another PR for that.